ui/app/models/task-event.js

```diff
@@ -48,100 +27,4 @@ export default Fragment.extend({
   vaultError: attr('string'),
   message: attr('string'),
   failedSibling: attr('string'),
```

Comment:
Looking at the Nomad source code, `vaultError` and `failedSibling` are also safe to remove: https://github.com/hashicorp/nomad/blob/master/nomad/structs/structs.go#L3979-L3986

Basically all of these specific properties were moved into a bag of properties named `Details`. Until `Details` is used in the application, it doesn't need to be added to the UI model.

Reply:
Cool, I've removed `vaultError` and `failedSibling`, also searched for `Passthru` and removed anything I could find that had anything to do with messagePassthru. I think I've got everything